help/sources/tutorials/ui/create/marketing-automation/braze.md

```diff
@@ -0,0 +1,58 @@
+---
+keywords: Experience Platform;home;popular topics;braze;Braze,Currents
+solution: Experience Platform
+title: Create a Braze Source Connection in the UI
+type: Tutorial
+description: Learn how to create a Braze source connection using the Adobe Experience Platform UI and Braze Currents
+exl-id: 452b7290-b9e8-4728-8b58-0e0c76bd9449
+---
+# Create a [!DNL Braze] source connection in the UI
+
+Source connectors in Adobe Experience Platform provide the ability to ingest externally sourced data on a scheduled basis. This tutorial provides steps for creating a streaming [!DNL Braze] source connector using the [!DNL Platform] user interface.
+
+## Getting started
+
+This tutorial requires a working understanding of the following components of Adobe Experience Platform:
+
+*   [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+    *   [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+    *   [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+*   [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+This tutorial also requires a working understanding of [[!Braze Currents]https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents).
+
+If you already have a [!DNL Braze] connection, you may skip the remainder of this document and proceed to the tutorial on [configuring a dataflow](../../dataflow/marketing-automation.md).
+
+## Create your [!DNL Platform] Data Source
+
+Log in to [Adobe Experience Platform](https://platform.adobe.com) and then select **[!UICONTROL Sources]** from the left navigation bar to access the **[!UICONTROL Sources]** workspace. The **[!UICONTROL Catalog]** screen displays a variety of sources from which you can select.
+
+Select the Marketing Automation category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the **[!UICONTROL Marketing automation]** category, select **[!UICONTROL Braze]**. If this is your first time using this connector, select **[!UICONTROL Configure]**. Otherwise, select **[!UICONTROL Add data]** to create a new [!DNL Braze] connector.
+
+![catalog](../../../../images/tutorials/create/braze/catalog.png)
```

Comment:
Let's update the alt-text here from "catalog" to "The sources catalog on the Experience Platform UI with the Braze Currents source selected."